.clang-format

```diff
@@ -0,0 +1,3 @@
+BasedOnStyle: Mozilla
+IndentWidth: 2
+ColumnLimit: 100
```

Comment:
Please consider increasing this to 110 or so to avoid reformatting all the copyright comments and other weird/excessive line breaks. Or consider switching to SPDX (which we should do regardless).

[Here](https://gerrit.named-data.net/c/NFD/+/3244/11/.clang-format) I kept a limit of 100 but I also played with the penalties to soften the limit and make line breaks happen in more reasonable places (still WIP though).

Reply:
Bumped to 110